### PR TITLE
feat(domain): move to github pages

### PR DESCRIPTION
**Pull request by itsmenewbie03**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel or Netlify.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.  

## Website Link/Preview
https://itsmenewbie03.github.io/

### CODE REVIEW

File: domains/itsmenewbie03.json [hunk 1]
```diff
@@ -4,6 +4,6 @@
         "email": "2101102516@student.buksu.edu.ph"
     },
     "record": {
-        "A": ["46.250.251.63"]
+        "CNAME": "itsmenewbie03.github.io"
```
Comment (CuteDog5695) [inline domains/itsmenewbie03.json:7]: No, just no. We do not allow NSFW

**Review by iostpa — CHANGES_REQUESTED** [review]

**Comment by VaibhavSys** [comment]
😭😭😭

**Comment by CuteDog5695** [comment]
Also maintainers are minors, so well done on giving us hentai you fucking degenerate

**Comment by itsmenewbie03** [comment]
> Also maintainers are minors, so well done on giving us hentai you fucking degenerate

Sorry that was a joke meant for my friends, didn't meant any harm.

**Pull request closed without merging**